sqlmesh/utils/metaprogramming.py

```diff
@@ -360,6 +361,7 @@ def serialize_env(env: t.Dict[str, t.Any], path: Path) -> t.Dict[str, Executable
                     # Do `as_posix` to serialize windows path back to POSIX
                     path=t.cast(Path, file_path).relative_to(path.absolute()).as_posix(),
                     alias=k if name != k else None,
+                    is_metadata=getattr(v, c.SQLMESH_METADATA, None),
```

Comment:
Can the default be `False` instead of `None`?

Reply:
I guess `is_metadata` would still have to be `Optional` so that the existing state can be deserialized. So doesn't matter.